applications/rubintv/templates/deployment-frontend.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "rubintv.fullname" . }}-frontend
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "rubintv.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: "Recreate"
```

Comment:
Why this? Normally you don't want this, since it forces an outage whenever the application is upgraded.

Reply:
Probably pasted from the existing rubintv?  And I don't know why.  But let's drop it.